### ISSUE TITLE
Working in melodic is partially successful. But I want to ask the question about xacro.

### ISSUE THREAD

**Issue by Yoo-Youngjae** [simlabrobotics/allegro_hand_ros_v4#2]
Hello. 
I am Yoo, a robot developer, who has run this allegro_hand_ros_v4 on melodic. 
After some debugging, I succeeded in running allegrohand v4 on ubuntu 18.04 (melodic). 
But I have some problems, so I want to ask.
First, it was successful until catkin_make. 

However, an error occurred during the roslaunch running and it did not work. 
The cause of the error is the following file: https://github.com/simlabrobotics/allegro_hand_ros_v4/blob/master/src/allegro_hand_description/allegro_hand_description_right.xacro

In conclusion, I solved this by annotating lines 87 through 117 in this file. 
This is because an error occurred that the finger and thumb_right of xacro could not be found. 
However, the hand connected through pcan-usb succeeded in performing the operation, but there is an error that the finger does not appear on the rviz.

How can I fix this? 
Could there be a problem with the real allegro hand in this situation?
I look forward to your advice.

Thanks.

**Comment by DavidYaonanZhu** [comment]
Same problem

**Comment by lihaosheng191240022** [comment]
same problem

**Comment by Ericcsr** [comment]
same problem

**Comment by JJMa1907** [comment]
reference https://github.com/simlabrobotics/allegro_hand_ros_catkin/pull/2#issue-881477912
